docs/errorhandling.rst

```diff
@@ -28,6 +28,59 @@ exception to the :attr:`~flask.Flask.logger`.
 But there is more you can do, and we will cover some better setups to deal
 with errors.
 
+.. _error-handlers:
+
+Error handlers
+--------------
+
+.. versionadded:: 1.0
+
+Some errors are unavoidable because not only you make them.  Users might
```

Comment:
hmm, my intention was:

1. general introduction: things go wrong, this is what can happen, now some ways to :sunglasses: deal with it
2. in some cases you might want to show error pages, this is when and how: …
3. if the default logging isn’t enough, do this: …
4. …